pages/common/vue-init.md

```diff
@@ -0,0 +1,18 @@
+# vue-init
```

Comment:
Could we change the title so that it matches the format of other subcommand pages like `git merge`, please?
```suggestion
# vue init
```